### PR TITLE
Update plugins

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -68,12 +68,6 @@ subprojects {
          * https://github.com/johnrengelman/shadow/blob/master/build.gradle
          */
 
-        // We need to do this otherwise the shadowJar tasks takes forever since
-        // it tries to shadow the entire gradle api
-        // Moreover, the gradleApi and groovy dependencies in the plugins themselves
-        // have been explicitly declared with the shadow configuration
-        configurations["compile"].dependencies.remove(dependencies.gradleApi())
```
Comment (rnc) [inline build.gradle.kts]: @geoand Usefully the latest version of the shadow plugin does this automatically :-)

Comment (geoand) [reply build.gradle.kts]: That's great! The less magic configuration the better!

**Comment by geoand** [comment]
Did CI not run perhaps?

**Comment by rnc** [comment]
@geoand Its a lazy ci :-) wakes up eventually !

**Comment by geoand** [comment]
:laughing:

**Pull request merged**